Description: Correct the path for starting direwolf in the example start script
Author: Iain R. Learmonth <irl@fsfe.org>
Last-Update: 2014-09-08

--- direwolf.orig/dw-start.sh
+++ direwolf/dw-start.sh
@@ -62,13 +62,13 @@
 
 if [ -x /usr/bin/lxterminal ]
 then
-  /usr/bin/lxterminal -t "Dire Wolf" -e "/usr/local/bin/direwolf" &
+  /usr/bin/lxterminal -t "Dire Wolf" -e "/usr/bin/direwolf" &
 elif [ -x /usr/bin/xterm ] 
 then
-  /usr/bin/xterm -bg white -fg black -e /usr/local/bin/direwolf &
+  /usr/bin/xterm -bg white -fg black -e /usr/bin/direwolf &
 elif [ -x /usr/bin/x-terminal-emulator ]
 then
-  /usr/bin/x-terminal-emulator -e  /usr/local/bin/direwolf &
+  /usr/bin/x-terminal-emulator -e  /usr/bin/direwolf &
 else
   echo "Did not find an X terminal emulator."
 fi
